pkg/build/api/v1/types.go

```diff
@@ -477,6 +477,13 @@ type SourceBuildStrategy struct {
 
 	// forcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty"`
+
+	// RuntimeImage is the optinal image that is used to run an application
+	// without unneeded dependencies installed.
+	RuntimeImage *kapi.ObjectReference `json:"runtimeImage,omitempty"`
+
+	// RuntimeArtifacts holds the mapping between the artifacts paths on the builder image and the paths on the runtime image where they should be copied.
+	RuntimeArtifacts []ImageSourcePath `json:"runtimeArtifacts,omitempty"`
```

Comment:
Yes, I was talking about adding another field to `Config`. Right now I'm re-using the same `Docker` instance that we're using for downloading builder image (with the same pull secrets).

As far I understand OpenShift doesn't download images in this case, right?